README.md

```diff
@@ -186,24 +183,24 @@ export const parameters = {
 }
 ```
 
-| sourceLinkPrefix in component | sourceLink in component | sourceLinkPrefix in story | sourceLink in story |           link                  |
-|:-----------------------------:|:-----------------------:|:-------------------------:|:-------------------:|:-------------------------------:|
-|               ✔               |            ✔            |             ✔             |          ✔          | story-prefix:story-link         |
-|               ✔               |            ✔            |             ✔             |                     | story-prefix:component-link     |
-|               ✔               |            ✔            |                           |          ✔          | component-prefix:story-link     |
+| sourceLinkPrefix in component | sourceLink in component | sourceLinkPrefix in story | sourceLink in story |              link               |
```

Comment:
This is just my VSCode being silly when I edit Markdown files with tables. Can revert if you want.